warn: deprecated packages

See sample output

```
npm warn exec The following package was not found and will be installed: zettel-lint@0.11.5
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
node:internal/modules/esm/resolve:26[5](https://github.com/craignicol/journal/actions/runs/10802354076/job/29964240300#step:6:6)
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/runner/.npm/_npx/d4bb02629f7d9b1a/node_modules/zettel-lint/lib/zl-index' imported from /home/runner/.npm/_npx/d4bb02629f7d9b1a/node_modules/zettel-lint/lib/zl.js
    at finalizeResolution (node:internal/modules/esm/resolve:2[6](https://github.com/craignicol/journal/actions/runs/10802354076/job/29964240300#step:6:7)5:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:540:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:509:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:239:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:96:40)
    at link (node:internal/modules/esm/module_job:95:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///home/runner/.npm/_npx/d4bb02629f[7](https://github.com/craignicol/journal/actions/runs/10802354076/job/29964240300#step:6:8)d9b1a/node_modules/zettel-lint/lib/zl-index'
}

Node.js v20.17.0
Error: Process completed with exit code 1.```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zettel-lint",
-  "version": "0.11.5",
+  "version": "0.11.6",
   "description": "Command line zettel linting/indexing",
   "main": "./lib/zl.js",
   "type": "module",
@@ -14,8 +14,7 @@
     "test": "jest",
     "help": "npm run-script zl -- --version && npm run-script zl -- --help",
     "refresh": "rm -rf ./node_modules ./package-lock.json && npm install",
-    "zl": "tsx src/zl.ts",
-    "postbuild": "copyfiles -u 1 src/**/*.mustache lib"
+    "zl": "tsx src/zl.ts"
   },
   "repository": {
     "type": "git",
@@ -39,7 +38,6 @@
     "chalk": "^5.3.0",
     "clear": "^0.1.0",
     "commander": "^12.0.0",
-    "copyfiles": "^2.4.1",
     "figlet": "^1.7.0",
     "fs": "0.0.2",
     "glob": "^11.0.0",

--- a/src/zl.ts
+++ b/src/zl.ts
@@ -10,7 +10,7 @@ import chalk from "chalk";
 // THIS import is outside `src/` folder so fails the build
 // import {version as packageVersion} from "../package.json";
 
-export const version = "0.11.5";
+export const version = "0.11.6";
 
 var program = new commander.Command("zl");
 

